lib/Zonemaster/Backend/Config.pm

```diff
@@ -809,7 +809,7 @@ sub new_PM {
                 }
             }
         },
-        1
+        0.125
```

Comment:
Isn't that unrelated to the purpose of this PR?

Reply:
Indeed. Would you suggest to do this change in a different PR?

Reply:
Yes, and there some comment what the change is for. I saw it in #1094 too.

Reply:
This has been removed from this PR.